src/modules/item/sagas.ts

```diff
@@ -443,14 +443,15 @@ export function* itemSaga(legacyBuilder: LegacyBuilderAPI, builder: BuilderClien
   function* handleSetCollection(action: SetCollectionAction) {
     const { item, collectionId } = action.payload
     const newItem = { ...item }
+    const address: string = yield select(getAddress)
     if (collectionId === null) {
       delete newItem.collectionId
     } else {
       newItem.collectionId = collectionId
     }
     yield put(saveItemRequest(newItem, {}))
     yield take(SAVE_ITEM_SUCCESS)
-    yield put(closeModal('AddExistingItemModal'))
```

Comment:
This `closeModal` is not required because the module: `modal.sagas` takes the action: `SAVE_ITEM_SUCCESS` and execute the method: `handleCloseAllModals`.
https://github.com/decentraland/builder/blob/master/src/modules/modal/sagas.ts#L13